plugins/reporters/web-app-template/src/models/WebAppRuleViolation.js

```diff
@@ -104,6 +122,12 @@ class WebAppRuleViolation {
                 }
             }
 
+            this.#isResolved = !!(this.#resolutionIndexes.size > 0);
+
+            if (this.#isResolved) {
+                this.#severityIndex = this.#severityIndex + 10;
```

Comment:
Where does the "magic number" 10 come from here?

Reply:
Could have been any other number as long as it bigger than 3 but 10 allows for addition of future states and made things easier for remember e.g.  SeverityIndex < 10 = unresolved error, warning or hint policy rule violation / issue, anything > 10 is resolved error, warning or hint policy rule violation / issue. Why the "bigger than 3" well you have to take into account column sorting which operates on comparing SeverityIndex integer instead of strings (Error, Warning, Hint).

- Error - SeverityIndex 1
- Warning - SeverityIndex 2
- Hint - SeverityIndex 3
- Resolved Error - SeverityIndex 11
- Resolved Warning - SeverityIndex 12
- Resolved Hint - SeverityIndex 13